Shared/Objects/VideoPlayerType/VideoPlayerType+Swiftfin.swift

```diff
@@ -137,4 +137,124 @@ extension VideoPlayerType {
             SubtitleFormat.xsub
         }
     }
+
+    // MARK: - Codec Profiles
+
+    @ArrayBuilder<CodecProfile>
+    static var _swiftfinCodecProfiles: [CodecProfile] {
+        CodecProfile(
```

Comment:
Use the init as defined in `CodecProfile` with the `ArrayBuilder` parameters for the profiles in this file.